root/static/styles/layout.less

```diff
@@ -18,6 +18,7 @@ body {
 a {
     color:@link-default;
     text-decoration: none;
+    word-break: keep-all;
```

Comment:
`word-break: break-all;` should be a better choice here, IMO. works in most browsers, including webkit-based ones